app/controllers/users.js

```diff
@@ -158,6 +158,18 @@ exports.showFollowing = (req, res) => {
   showFollowers(req, res, 'following');
 };
 
+exports.delete = (req, res)=> {
```

Comment:
We leave a space before =>
So it should be something like `exports.delete = (req, res) => {`. There is nothing wrong with your code, just that we try to maintain a uniform coding style.